fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/security/KeyPair.java

```diff
@@ -39,10 +41,22 @@ public PublicKey getPublicKey() {
     return publicKey;
   }
 
+  @NonNull
+  @Override
+  public String toString() {
+    return getClass().getSimpleName()
+        + "{privateKey="
+        + privateKey.toString()
```

Comment:
I'm not sure it's a great idea to print the private key _anywhere_

Reply:
Good point

Now, for debug purpose, I think it should be great to see whether it is a plain or a protected key.
I think I will keep this but redeclare the `toString` of `PlainPrivateKey` to hide it if it is ever printed.

Reply:
I pushed a change on PlainPrivateKey, it should fix this isssue.